CircuitPython_Slider/code.py.py

```diff
@@ -0,0 +1,347 @@
+import time
```

Comment:
I noticed this file is named code.py.py instead of code.py. You might want to change that.